### ISSUE TITLE
day-08

### ISSUE THREAD

**Issue by VV20192019** [nickb-/tensorflow-probability-100days#1]
TFP Version 0.16.0
TF  Version 2.8.0

To obtain the correct results, the training data X must input all dimensions, and the author only inputs one dimension, which cannot reproduce the results in the graph

x_train_con = np.concatenate([X[0:200, :], xtrain[0:200][:, None]], axis=-1)
x_test_con = np.concatenate([X[200:250, :], xtest[:, None]], axis=-1)
model.compile(optimizer=tf.keras.optimizers.Adam(learning_rate=0.05), loss=negloglik)
model.fit(x_train_con, ytrain, epochs=500, verbose=1)

**Comment by nickb-** [comment]
Ah this is terrible practice by me, I haven't pinned my package versions! Oh no.

Thank you for your issue! I'm not 100% sure which part of the notebook you are referencing. Can you please point me to the exact cell/line?
